spec/categories_page_spec.rb

```diff
@@ -0,0 +1,72 @@
+    transaction9 = Entity.create({ name: 'Street pop corn', created_at: '2021-01-01', updated_at: '2021-01-01', user_id: user1.id, amount: 3.50 })
+    GroupEntity.create({ created_at: '2021-01-01', updated_at: '2021-01-01', group_id: group1.id, entity_id: transaction1.id })
+    GroupEntity.create({ created_at: '2021-01-01', updated_at: '2021-01-01', group_id: group1.id, entity_id: transaction2.id })
+    GroupEntity.create({ created_at: '2021-01-01', updated_at: '2021-01-01', group_id: group1.id, entity_id: transaction3.id })
+    GroupEntity.create({ created_at: '2021-01-01', updated_at: '2021-01-01', group_id: group1.id, entity_id: transaction7.id })
+    GroupEntity.create({ created_at: '2021-01-01', updated_at: '2021-01-01', group_id: group1.id, entity_id: transaction9.id })
+    GroupEntity.create({ created_at: '2021-01-01', updated_at: '2021-01-01', group_id: group2.id, entity_id: transaction4.id })
+    GroupEntity.create({ created_at: '2021-01-01', updated_at: '2021-01-01', group_id: group2.id, entity_id: transaction5.id })
+    GroupEntity.create({ created_at: '2021-01-01', updated_at: '2021-01-01', group_id: group2.id, entity_id: transaction6.id })
+    GroupEntity.create({ created_at: '2021-01-01', updated_at: '2021-01-01', group_id: group3.id, entity_id: transaction6.id })
+    GroupEntity.create({ created_at: '2021-01-01', updated_at: '2021-01-01', group_id: group3.id, entity_id: transaction7.id })
+    GroupEntity.create({ created_at: '2021-01-01', updated_at: '2021-01-01', group_id: group3.id, entity_id: transaction8.id })
+    GroupEntity.create({ created_at: '2021-01-01', updated_at: '2021-01-01', group_id: group3.id, entity_id: transaction9.id })
+  end
+
+  describe 'I can see' do
+    it 'the splash screen' do
+      visit splash_show_path
+      expect(page).to have_link('Log in')
+      expect(page).to have_link('Sign up')
+    end
+
+    it 'the categories screen' do
+      visit splash_show_path
+      click_link 'Log in'
+      fill_in 'E-mail', with: 'user1@mail.com'
+      fill_in 'Password', with: '111111'
+      click_button 'Log in'
+      expect(page).to have_link('ADD NEW CATEGORY')
+    end
+
+    it 'the add new category screen' do
+      visit splash_show_path
+      click_link 'Log in'
+      fill_in 'E-mail', with: 'user1@mail.com'
+      fill_in 'Password', with: '111111'
+      click_button 'Log in'
+      click_link 'ADD NEW CATEGORY'
+      expect(page).to have_button('CREATE')
+    end
+
+    it 'that it creates a category succesfully' do
+      visit splash_show_path
+      click_link 'Log in'
+      fill_in 'E-mail', with: 'user1@mail.com'
+      fill_in 'Password', with: '111111'
+      click_button 'Log in'
+      click_link 'ADD NEW CATEGORY'
+      fill_in 'Name', with: 'new category'
+      fill_in 'Place your icon url here', with: 'https://lorempixel.com/200/200/food/'
+      click_button 'CREATE'
+      expect(page).to have_content('new category')
+    end
+  end
+end
```

Comment:
- [ ] Optional suggestion

You don't need to visit URL more than once in the test. You can visit the URL once and click the link or buttons.